### ISSUE TITLE
Make vuepress based docs

### ISSUE THREAD

**Issue by insani7y** [community-of-python/microbootstrap#12]
Vuepress is a framework that helps you build readme based documentation.
Search has to be implemented

**Comment by Lancetnik** [comment]
I don't think, that is a good idea. Vuepress / Docsaurus or another JS-based documentation engines are much harder to contribute for most python developers. It is a bad choice for python-first Open Source tool if you are welcome for documentation contribution from community. Mkdocs / sphinx are common choices here.

**Comment by vrslev** [comment]
Agreed. Well, I'm not sure such documentation is actually needed. A directory docs/ in code is sufficient for our use case
